backend/tests/unit/fake_elastic.py

```diff
@@ -31,6 +31,7 @@ class FakeAsyncElasticsearch(AsyncElasticsearch):
     args: dict[str, Any]
     closed: bool
     requests: list[Request]
+    requests: list[Request]
```

Comment:
The `requests` declaration got duplicated here. I remember seeing that in merging some of my draft PRs, too -- no idea why that happens.